versionhistory.md

```diff
@@ -0,0 +1,77 @@
+# Version history endpoints
+
+[Back to the list of all defined endpoints](endpoints.md)
+
+This endpoint represents all related versions.
+
+Whether version history information is accessible depends on versioning.item.history.view.admin configuration
+If this is set to true, the version history can only be retrieved if the user is an admin of the last version's item
+
+## Get version history
+
+**GET /api/versioning/versionhistory/<:versionHistoryId>**
+
+Provide version information for the version history id.
+
+```json
+{
+  "id": "1",
+  "type": "versionhistory",
+  "_links": {
+    "self": {
+      "href": "https://dspace7.4science.it/dspace-spring-rest/api/versioning/versionhistory/1"
+    },
+    "versions": {
+      "href": "https://dspace7.4science.it/dspace-spring-rest/api/versioning/versionhistory/1/versions"
+    }
+  }
+}
+```
+
+Status codes:
+* 200 OK - if the version history exists and is accessible by the current user
+* 401 Unauthorized - if you are not authenticated and versioning is not public
+* 403 Forbidden - if you are not logged in with sufficient permissions and versioning is not public
+* 404 Not found - if the version history doesn't exist
+
+## Linked entities
+
+### Versions
+
+**GET /api/versioning/versionhistory/<:versionHistoryId>/versions**
+
+Retrieve a pageable list of versions for the provided version history identifier.
```

Comment:
it will be useful to state in which order the versions are returned. It should be based on the creation time my preference go for descending order.

Reply:
I've included the detail about the order currently used in the Service. It returns the most recent first. The Service will need to be updated to specify this order in the JavaDocs